recipes/nbpresent/run_test.sh

```diff
@@ -0,0 +1,11 @@
+
+# On circle-ci, we are building in a container, so CIRCLE_CI is not set
+# So this will have to do.
+if [ `which yum` ]; then
+  yum install -y fontconfig freetype
```

Comment:
We have both fontconfig and freetype in conda-forge. Also, we discourage the use of yum in the build.sh. We have a yum_requires scheme for things like that.

I am on my phone, so I'll point you to an example later b/c I could not find it easily now.